packages/lib/gpc/src/gpcCompile.ts

```diff
@@ -421,6 +445,13 @@ function compileProofListMembership<
       paramMaxLists,
       BABY_JUB_NEGATIVE_ONE
     ),
+    // Pad with 1s, which amounts to a list membership check for those values corresponding to index
+    // -1 (which corresponds to the value 0).
```

Comment:
I think this line is copypasta which doesn't apply here.  There's no use of -1.

Reply:
There is actually. The padding here will agree with the padding directly preceding it, which is where the -1 comes in. I'll add a 'cf. ...' to clarify.